tests/mock-transport.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+class WP_Http_Mock_Transport {
+
+	public static $response = array();
+	public static $test_class;
+	public static $expected_url;
+
+	public static function test() {
+		return true;
```

Comment:
WordPress calls this method in order to determine if the transport is ready. Simply - it does not work without the method.